man/systemd.netdev.xml

```diff
@@ -822,6 +824,31 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[BareUDP] Section Options</title>
+
+    <para>The [BareUDP] section only applies for
+    netdevs of kind <literal>bareudp</literal>, and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>DestinationPort=</varname></term>
+        <listitem>
+          <para>Specifies destination port. This is mandatory. Default to unset.</para>
```

Comment:
specify range. maybe mention again this is an UDP port, here, i.e. `Specifies the destination UDP port (in range 1…65535).`

Reply:
defaults to unset when mandatory? It doesn't default to anything, i.e. drop the "Default to unset"

Reply:
opps yeah sorry